### PR TITLE
chore: try_into to into

### PR DESCRIPTION
**Pull request by u2**
https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=3cc77c2bf9357b5328b9627d0f2ff6ef

### CODE REVIEW

File: miner/src/block_assembler.rs [hunk 1]
```diff
@@ -538,7 +538,7 @@ mod tests {
     use ckb_verification::{BlockVerifier, HeaderResolverWrapper, HeaderVerifier, Verifier};
     use jsonrpc_types::{BlockTemplate, CellbaseTemplate};
     use numext_fixed_hash::H256;
-    use std::convert::TryInto;
+    use std::convert::Into;
```
Comment (yangby-cryptape) [inline miner/src/block_assembler.rs]: Unless we use `no_std`, this is unnecessary. (not a request change, just a comment)

File: miner/src/block_assembler.rs [hunk 2]
```diff
@@ -614,28 +614,10 @@ mod tests {
...
-                    .unwrap(),
-            )
-            .proposals(
-                proposals
-                    .into_iter()
-                    .map(TryInto::try_into)
-                    .collect::<Result<_, _>>()
-                    .unwrap(),
-            )
+            .uncles(uncles.into_iter().map(Into::into).collect())
```
Comment (yangby-cryptape) [inline miner/src/block_assembler.rs]: (not a request change, just a comment)

```rust
impl BlockBuilder {
    fn uncles<T>(mut self, arg: Vec<T>) -> Self where T: Into<UncleBlock>;
    fn uncle<T>(mut self, arg: T) -> Self where T: Into<UncleBlock>;
}
```

Comment (TheWaWaR) [reply miner/src/block_assembler.rs]: I prefer (like std does):
``` rust
impl BlockBuilder {
    fn uncles<T: Into<UncleBlock>>(mut self, arg: Vec<T>) -> Self;
    fn uncle<T: Into<UncleBlock>>(mut self, arg: T) -> Self;
}
```

Comment (yangby-cryptape) [reply miner/src/block_assembler.rs]: > I prefer (like std does):
> 
> ... ...

How many or how long you will change to the `where` statement?

Try to run `rustfmt` for this:
```rust
impl BlockBuilder {
    fn new<H: Into<Header>, U: Into<UncleBlock>, T: Into<Transaction>, O: Into<Something>>(mut self, header: H, uncles: Vec<U>, transactions: Vec<T>, others: O) -> Self {}
}
```

So, why do not use a consistent syntax?

Comment (u2) [reply miner/src/block_assembler.rs]: update

**Pull request merged**